src/components/footer-list/FooterList.jsx

```diff
@@ -1,27 +1,19 @@
 import { Box } from "@mui/material"
 import { List, ListItemButton, ListSubheader, ListItemText } from '@mui/material';
+import styles from "./FooterList.module.css";
 
-export const FooterList = ({ title, dataArr, type }) => {
-    const styles = {
-        lineHeight: "18px",
-        fontSize: 14,
-        fontWeight: 500,
-        textTransform: "capitalize",
-        paddingInline: "0px",
-        paddingBlock: "4px",
-        width: "100%"
-    }
+export const FooterList = ({ title, dataSource, type }) => {
```

Comment:
you just changed arr to source , what is this data ? buttons ? names ? ....etc I want to know what it is from the param name